app/components/logining/login.controller.js

```diff
@@ -0,0 +1,24 @@
+;(function () {
+  angular
+    .module("todo")
+    .controller("LoginController", LoginController);
+
+  LoginController.$inject = ["SessionService"];
+
+  function LoginController(SessionService) {
+    var self = this;
+    self.signIn = signIn;
+    self.isMsgHide = true;
+    self.loginError = loginError;
+    self.user = {};
```

Comment:
It's not right to initialize objects with empty object. If it's no data - `null` is the best option